.travis.yml

```diff
@@ -18,6 +18,9 @@ script:
   - bundle exec rspec spec
   - for f in examples/*.sh.txt; do echo "checking bash syntax $f"; bash -n $f; done
 
+after_success:
+- codeclimate-test-reporter
```

Comment:
Ack, this should probably be `bundle exec`'d.  This was previously being done in-process, but that way is no longer supported.